phaseshifts/factories.py

```
@@ -0,0 +1,82 @@
+    from typing import List  # noqa: F401
+except ImportError:
+    pass
+
+from .wrappers import BVHWrapper, EEASiSSSWrapper
+
+
+class PhaseshiftFactory(object):
+    """Class for backend selection"""
+
+    backend = object
+    phsh_files = []  # type: List[str]
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)